SimulationAgent.Test/SimulationManagerTest.cs

```diff
@@ -269,43 +270,71 @@ void ItCalculatesStatsAndInvokesCreateOrUpdateStatistics()
             var expectedFailedMessagesCount = 2;
             var expectedFailedDeviceConnectionsCount = 6;
             var expectedFailedTwinUpdatesCount = 10;
+            var expectedActiceDevicesCount = 2;
```

Comment:
rename: there is a typo here 'expectedActiveDevicesCount '

Reply:
fixed